src/Models/Location.php

```diff
@@ -146,25 +151,24 @@ public function getSelectorTitleAttribute()
             return "{$this->market->state} - {$this->name}";
         }
     }
-    
+
     public function getContactEmailAddressAttribute()
     {
-        return $this->contact_email->email;
+        return $this->contactEmail()->email;
     }
 
     public function getStreetAddressAttribute()
     {
         $add2 = '';
-        if ($this->address2) {
-            $add2 = ' ' . $this->address2;
+        if ($this->address()->address_line_2) {
+            $add2 = ' ' . $this->address()->address_line_2;
         }
-
-        return "{$this->address}{$add2}";
+        return "{$this->address()->address_line_1}{$add2}";
     }
 
     public function getFullAddressAttribute()
     {
-        return "{$this->street_address}, {$this->city}, {$this->state} {$this->zip}";
+        return "{$this->address()->address_line_1}, {$this->address()->city}, {$this->address()->zip}";
```

Comment:
Does domestic address model track state?

Reply:
Yes, but it will be either through the City (https://github.com/tipoff/addresses/issues/18) or ZIP Code (https://github.com/tipoff/addresses/issues/17). Let's use ZIP Code to fetch the State (https://github.com/tipoff/addresses/issues/16)